src/python/pants/backend/go/README.md

```diff
@@ -0,0 +1,40 @@
+# GoLang Backend - Under Development!
```

Comment:
Thanks for documenting this! That being said, it's probably fine to leave this off and consolidate the status quo in the relevant GitHub issue. The `experimental` prefix already expresses this disclosure. It's better to track the remaining TODOs in that ticket than here too.

Reply:
I've removed the list of tasks, but I left in remainder of the README because it makes clear that "experimental" does not mean "ready for even development use."